### PR TITLE
Created new view where jackpot can be updated

### PR DESCRIPTION
**Pull request by MiguelTrujillo26**
Jackpot edit view

### CODE REVIEW

File: web/src/utils/componentUtil.js [hunk 1]
```diff
@@ -29,8 +29,19 @@ const fileToBase64 = async (file) => {
 
 const hideElement = (event) => event.target.style.display = "none";
 
+const validateImageFile = (file) => {
+    const validImageTypes = ['image/jpeg', 'image/png'];
```
Comment (andymotagl) [inline web/src/utils/componentUtil.js:33]: Let's admit any image at the moment.

File: web/src/utils/componentUtil.js [hunk 2]
```diff
@@ -29,8 +29,19 @@ const fileToBase64 = async (file) => {
...
+    const maxSize = 1024 * 1024;
+    if(!file)  {return 'Imagen no seleccionada'};
+    console.log(file.type);
```
Comment (andymotagl) [inline web/src/utils/componentUtil.js:36]: Remove

**Review by sebastian-motavita — APPROVED** [review]
LGTM but, consider my comment

**Review by sebastian-motavita — APPROVED** [review]
LGTM but consider my comments

File: web/src/views/EditJackpot.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+import { BACKEND_URL } from '../utils/constants';
+import {
+    fileToBase64,
+    validateImageFile
+} from '../utils/componentUtil';
+
+const EditJackpot = () => {
+    const { error, getAccessTokenSilently } = useAuth0();
+    const {id} = useParams();
+    const [jackpot, setJackpot] = useState({name:'',description:'',image:''});
```
Comment (sebastian-motavita) [inline web/src/views/EditJackpot.js:28]: Just nitpicking: format a little better.

**Pull request merged**